pkg/cmd/extension/manager.go

```diff
@@ -274,9 +274,11 @@ func (m *Manager) installBin(repo ghrepo.Interface, target string) error {
 	}
 
 	if asset == nil {
+		issueCreateCommand := generateMissingBinaryIssueCreateCommand(repo.RepoOwner(), repo.RepoName(), platform)
+
 		return fmt.Errorf(
-			"%[1]s unsupported for %[2]s. Open an issue: `gh issue create -R %[3]s/%[1]s -t'Support %[2]s'`",
-			repo.RepoName(), platform, repo.RepoOwner())
+			"%[1]s unsupported for %[2]s. Open an issue: `%[3]s`",
+			repo.RepoName(), platform, issueCreateCommand)
```

Comment:
I'm finding this really hard to parse in the terminal as it's just a blob of text:

<img width="1511" alt="Screenshot 2024-09-12 at 11 30 31 AM" src="https://github.com/user-attachments/assets/e5c6e183-66a2-47a2-ba21-74f2454cc634">

We do have some style guidelines in [Primer](https://primer.style/native/cli) for the CLI we could leverage. I'm thinking that we can at least break this into a few lines and use a bit of color following those guidelines.

Mind taking a crack at it?

Reply:
That's a really nice idea.

I think at least breaking the command onto another line would be very beneficial and adding some indentation would be beneficial.

Reply:
@jtmcg Here's how it looks like in 414a59e:

![Screenshot 2024-09-12 at 23 19 21](https://github.com/user-attachments/assets/7d085640-642d-4e4f-928a-b0081589177f)

I'm open to doing more, but I'm a bit worried about how to do cross-platform line breaks within the command...